src/lib/handlebars-converter/handlebars-helpers.js

```diff
@@ -594,5 +594,81 @@ module.exports.external = [
         func: function (o) {
             return isNaN(o);
         }
+    },
+    {
+        name: 'abs',
+        description: 'Returns the absolute value of a number.',
+        func: function (x) {
+            return Math.abs(x);
```

Comment:
Should we throw a friendly error in case of exception?

Reply:
Discussed with Deepak, and we'll leave the math functions returning NaN if unable to calculate a numeric value.